content/en/docs/tasks/job/indexed-parallel-processing-static.md

```diff
@@ -96,10 +104,15 @@ like shown in the following example:
 Now run the Job:
 
 ```shell
-kubectl apply -f ./indexed-job.yaml
+# This uses the first approach (relying on $JOB_COMPLETION_INDEX)
+kubectl apply -f https://kubernetes.io/examples/application/job/indexed-job.yaml
 ```
 
-Wait a bit, then check on the job:
+When you create this Job, the control plane creates a series of Pods, one for each index you specified. The value of `.spec.parallelism` determines how many can run at once whereas `.spec.completions` determines how many Pods the Job creates in total.
```

Comment:
nit: The value of ... determines how many Pods (?) can run [immediately or at the same time]?